gcip/addons/container/jobs/dive.py

```diff
@@ -0,0 +1,100 @@
+__author__ = "Thomas Steinbach"
+__copyright__ = "Copyright 2020 DB Systel GmbH"
+__credits__ = ["Thomas Steinbach", "Daniel von Eßen"]
+# SPDX-License-Identifier: Apache-2.0
+__license__ = 'Apache-2.0'
+__maintainer__ = 'Daniel von Eßen'
+__email__ = 'daniel.von-essen@deutschebahn.com'
+
+from typing import List, Optional
+
+from gcip.core.job import Job
+from gcip.core.image import Image
+from gcip.core.variables import PredefinedVariables
+
+
+def _validate_float_for_scan(validate: float) -> bool:
```

Comment:
Just a suggestion

```suggestion
def _is_float_between_0_and_1(validate: float) -> bool:
```

or

```suggestion
def _is_float_between_zero_and_one(validate: float) -> bool:
```

Ask yourself what does "float for scan" mean, when not seeing the rest of this code? And then ask what does "float between zero and one" mean? Latter sounds more obvious (to me) :)